portafly/src/components/pages/accounts/listing/AccountsListingTable.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect } from 'react'
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  OnSelect,
+  OnSort
+} from '@patternfly/react-table'
+import {
+  PageEmptyState,
+  useDataListFilters,
+  useDataListPagination,
+  useDataListTable,
+  useDataListBulkActions,
+  Toolbar,
+  SearchWidget,
+  PaginationWidget,
+  BulkSelectorWidget,
+  SendEmailModal,
+  ChangeStateModal,
+  BulkActionsWidget,
+  filterRows
+} from 'components'
+import { useTranslation } from 'i18n/useTranslation'
+import { ToolbarContent, ToolbarItem } from '@patternfly/react-core'
+
+const AccountsListingTable: React.FunctionComponent = () => {
+  const { t } = useTranslation('audienceAccountsListing')
+  const {
+    columns,
+    rows,
+    sortBy,
+    setSortBy,
+    selectedRows,
+    selectOne
+  } = useDataListTable()
+
+  if (rows.length === 0) {
+    return <PageEmptyState msg={t('accounts_table.empty_state')} />
+  }
+
+  const { startIdx, endIdx, resetPagination } = useDataListPagination()
+  const { filters } = useDataListFilters()
+  const { modal } = useDataListBulkActions()
+
+  const options = [
+    { name: 'approved', humanName: t('actions_filter_options.by_state_options.approved') },
+    { name: 'pending', humanName: t('actions_filter_options.by_state_options.pending') },
+    { name: 'rejected', humanName: t('actions_filter_options.by_state_options.rejected') },
+    { name: 'suspended', humanName: t('actions_filter_options.by_state_options.suspended') }
+  ]
+
+  const categories = [
+    { name: 'group', humanName: t('accounts_table.group_header') },
+    { name: 'admin', humanName: t('accounts_table.admin_header') },
+    { name: 'state', humanName: t('accounts_table.state_header'), options }
+  ]
```

Comment:
I think it's good to have these options here, but will want to make sure these won't change give an account plan or some modifier. If that's the case, we will need to get them from the API